zkevm-circuits/src/state_circuit.rs

```diff
@@ -32,6 +32,9 @@ use random_linear_combination::{Chip as RlcChip, Config as RlcConfig, Queries as
 use std::collections::HashMap;
 use std::iter::once;
 
+// TODO: use a better value for production.
+const N_ROWS: usize = 1 << 16;
```

Comment:
Can this constant be moved to the `StateCircuit`?: `pub struct StateCircuit<F: Field, const N_ROWS: usize>`.  So that we can keep tests small and use higher value for production circuits.